acestep/ui/gradio/events/library_handlers.py

```
@@ -0,0 +1,195 @@
+            try:
+                ts = int(os.path.getmtime(native_path))
+            except Exception:
+                ts = 0
+
+            metadata: dict = {}
+            if os.path.exists(json_path):
+                try:
+                    with open(json_path, "r", encoding="utf-8") as f:
+                        metadata = json.load(f)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)